protos/bentoml_service.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
```

Comment:
This file should just be named `service.proto`, as this is bentoml spec only. Service is a core bentoml concept, hence no need for the distinction here.

Reply:
Done!